launcher/ui/pages/global/APIPage.cpp

```diff
@@ -36,12 +36,16 @@
  */
 
 #include "APIPage.h"
+#include <quuid.h>
+#include <qvalidator.h>
```

Comment:
Why is this including the internal Qt headers? This does not seem like a good idea, as far as I can see those are an implementation detail that could be changed at any time.

Reply:
That's just clangd or nvim-lspconfig being stupid.